a-short-introduction-to-kubernetes/index.md

```diff
@@ -0,0 +1,372 @@
+---
+title: 'Introduction to Kubernetes'
+description: 'A short introduction to Kubernetes in a Directus context'
+author:
+  name: 'Mikke Schirén'
+  avatar_file_name: 'mikke.png'
+---
+
+## Introduction
+
+Originated in Googles project Borg, which started in 2003, Kubernetes (K8s) is today the most popular open-source container orchestration system, and it automates Deployments, scaling and management of applications. And Kubernetes is in many ways, a complicated beast to handle, like a puzzle with many pieces. If you never used Kubernetes before, it is maybe not the easiest way forward for your self-hosted Directus project. But there a lot of gains with Kubernetes running an application like Directus.
+
+Good to remember that Kubernetes is not a traditional, all-inclusive PaaS (Platform as a Service) system, when you start up a new cluster, depending on which flavor your using, you need to install some, or all pieces yourself.
+
+If you never used Kubernetes, you may be surprised that you very often already have it on your development computer. Kubernetes, in a simple flavor is for instance shipped with Docker Desktop for Mac. So the place to test out Kubernetes itself it is very often nearby. If you want to experiment with Kubernetes for the first time, I really recommend that you do so locally.
+
+I set up my own Kubernetes cluster on Amazon some years ago - before they had their offering [EKS](https://aws.amazon.com/eks/) (Amazon Elastic Kubernetes Service). I learned a lot, but I would not recommend my worst enemy to do it. Today there are a lot of ready Kubernetes solutions for you to choose from - like EKS, [GKE](https://cloud.google.com/kubernetes-engine) (Google Kubernetes Engine), Microsoft [Azure Kubernetes Service](https://azure.microsoft.com/en-us/products/kubernetes-service) (AKS) etc. And if you want more control you can setup Kubernetes with [Rancher](https://www.rancher.com/) (which we mostly do these days).
```

Comment:
Haha this is funny but as an educational piece we don't need to jump into your personal gripes with different vendors. 

```suggestion
There are a lot of hosted Kubernetes solutions for you to choose from - like Amazon's [EKS](https://aws.amazon.com/eks/), Google's [GKE](https://cloud.google.com/kubernetes-engine), and Microsoft's [AKS](https://azure.microsoft.com/en-us/products/kubernetes-service). 
```

Beginner post, so no need to mention Rancher.